### ISSUE TITLE
icon file must exist: 'logo.icns' - error

### ISSUE THREAD

**Issue by kosmynkab** [simoncozens/fontbakery-ui#11]
While building through terminal with "setup.py" i got the following error with the repo synced to desktop.

**Comment by simoncozens** [comment]
That's weird. The file's definitely there: https://github.com/simoncozens/fontbakery-ui/blob/master/logo.icns

**Comment by kosmynkab** [comment]
Yeah! I know - I have the entire repo synced and that is the issue I get in terminal when I run the setup.py

on 10.15.2 OSX if that has any impact.
